app/response_processor.py

```diff
@@ -77,25 +90,21 @@ def _send_rm_receipt(self, case_id, user_id):
         try:
             r = self.session.post(request_url, auth=settings.BASIC_AUTH, json=request_json)
         except MaxRetryError:
-            logger.error("Max retries exceeded (5)",
-                         request_url=request_url,
-                         case_id=case_id)
+            self.logger.error("Max retries exceeded (5)",
```

Comment:
Why remove the case id from the log line?  We'd want as much information as possible